monai/bundle/utils.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import re
+from distutils.util import strtobool
+from typing import Any, Dict, Optional, Sequence, Tuple, Union
+
+from monai.bundle.config_parser import ConfigParser
+from monai.utils import ensure_tuple, optional_import
+
+yaml, _ = optional_import("yaml")
+
+
+def load_config_file(filepath: str, **kwargs):
+    """
+    Load config file with specified file path.
+    Suppprt JSON and YAML formats.
+
+    Args:
+        filepath: path of target file to load, supported postfixes: `.json`, `yml`, `yaml`.
+        kwargs: other arguments for `json.load` or `yaml.load`, depends on file format.
+            for more details, please check:
+            https://docs.python.org/3/library/json.html#json.load.
+            https://pyyaml.org/wiki/PyYAMLDocumentation.
+
+    """
+    with open(filepath) as f:
+        if filepath.lower().endswith(".json"):
+            return json.load(f, **kwargs)
+        if filepath.lower().endswith((".yml", ".yaml")):
+            if "Loader" not in kwargs:
+                kwargs["Loader"] = yaml.FullLoader
+            return yaml.load(f, **kwargs)
```

Comment:
Should use `yaml.safe_load` if we can, we're not using YAML's ability to construct Python objects with `!!python/object` syntax so it's safer to disallow it unless that creates issues.

Reply:
Sure, updated in the latest commit.

Thanks.